### PR TITLE
Added default contents for empty files plus minor changes (was PR #47)

### PR DESCRIPTION
**Pull request by almostearthling**
Hi, I just replicated the changes I made in the previous PR, applying one minor correction that was probably among the ones that caused the impossibility to build correctly. I tested the new script on one of my installations and it doesn't seem to generate errors. Some more tests shuld be done though.

Also, minor spell checks were added.

F.

### CODE REVIEW

**Comment by almostearthling** [comment]
Looks like Travis CI fails for the OAuth reason.

F.

**Comment by almostearthling** [comment]
Also, a spec fails as my modified version of `fileContent` returns null on empty/error:

```
  low-level
    ::fileContent
      it returns non-empty string for not existing file
        Expected null to be truthy. (spec/sync-settings-spec.coffee:15:73)
      it returns non-empty string for empty file
        Expected null to be truthy. (spec/sync-settings-spec.coffee:20:75)
```

Would you prefer to have an empty string instead?

F.

**Comment by groupsky** [comment]
Looks good. :+1:

**Comment by almostearthling** [comment]
Hope you like that expectations from `fileContent` can be modified in the specs, in order to have a simple way to handle the different file types in `upload`.

F.

File: lib/sync-settings.coffee [hunk 1]
```diff
@@ -42,17 +42,23 @@ module.exports =
...
     for file in atom.config.get('sync-settings.extraFiles') ? []
+      ext = file.slice(file.lastIndexOf(".")).toLowerCase()
+      cmtstart = "#"
+      cmtstart = "//" if ext in [".less", ".scss", ".js"]
+      cmtstart = "/*" if ext in [".css"]
+      cmtend = ""
+      cmtend = "*/" if ext in [".css"]
       files[file] =
-        content: @fileContent atom.config.configDirPath + "/#{file}"
+        content: (@fileContent atom.config.configDirPath + "/#{file}") ? "#{cmtstart} extra (not found) #{cmtend}"
```
Comment (jerone) [inline lib/sync-settings.coffee]: Can you replace `"#{cmtstart} extra (not found) #{cmtend}"` with `"#{cmtstart} {file} file (not found) #{cmtend}"`.

File: menus/sync-settings.cson [hunk 2]
```diff
@@ -3,7 +3,7 @@
   {
     'label': 'Packages'
     'submenu': [
-      'label': 'Syncronize Settings'
+      'label': 'Synchronize Settings'
```
Comment (jerone) [inline menus/sync-settings.cson]: :+1:

**Comment by almostearthling** [comment]
@jerone Of course. :+1:

F.

**Comment by jerone** [comment]
@groupsky I'm happy with merging this; my tests and Travis seems good...

**Comment by groupsky** [comment]
@jerone go for it :+1:

**Comment by jerone** [comment]
@almostearthling Thanks for the great PR! :+1:

**Comment by almostearthling** [comment]
Thank you all for the job! ;-)

F.

**Pull request merged**